pkg/ebpf/telemetry/errors_collector_linux.go

```diff
@@ -31,17 +31,26 @@ type EBPFErrorsCollector struct {
 	T                *EBPFTelemetry
 	ebpfMapOpsErrors *prometheus.Desc
 	ebpfHelperErrors *prometheus.Desc
+	lastValues       map[metricKey]uint64
+}
+
+type metricKey struct {
+	hash uint64
+	id   int
+	err  string ``
 }
 
 // NewEBPFErrorsCollector initializes a new Collector object for ebpf helper and map operations errors
 func NewEBPFErrorsCollector() prometheus.Collector {
 	if supported, _ := ebpfTelemetrySupported(); !supported {
 		return nil
 	}
+
 	return &EBPFErrorsCollector{
 		T:                newEBPFTelemetry(),
-		ebpfMapOpsErrors: prometheus.NewDesc(fmt.Sprintf("%s__errors", ebpfMapTelemetryNS), "Failures of map operations for a specific ebpf map reported per error.", []string{"map_name", "error"}, nil),
-		ebpfHelperErrors: prometheus.NewDesc(fmt.Sprintf("%s__errors", ebpfHelperTelemetryNS), "Failures of bpf helper operations reported per helper per error for each probe.", []string{"helper", "probe_name", "error"}, nil),
+		ebpfMapOpsErrors: prometheus.NewDesc(fmt.Sprintf("%s__%s__errors", ebpfPrefix, ebpfMapTelemetryNS), "Failures of map operations for a specific ebpf map reported per error.", []string{"map_name", "error"}, nil),
+		ebpfHelperErrors: prometheus.NewDesc(fmt.Sprintf("%s__%s__errors", ebpfPrefix, ebpfHelperTelemetryNS), "Failures of bpf helper operations reported per helper per error for each probe.", []string{"helper", "probe_name", "error"}, nil),
```

Comment:
This is the only place the name is used, so extracting constants seems unnecessary.
```suggestion
		ebpfMapOpsErrors: prometheus.NewDesc("ebpf__maps__errors", "Failures of map operations for a specific ebpf map reported per error.", []string{"map_name", "error"}, nil),
		ebpfHelperErrors: prometheus.NewDesc("ebpf__helpers__errors", "Failures of bpf helper operations reported per helper per error for each probe.", []string{"helper", "probe_name", "error"}, nil),
```